cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -963,6 +963,58 @@ public void testTableSegmentRecoveryCommand() throws Exception {
         Assert.assertArrayEquals(Files.readAllBytes(Paths.get(pravegaStorageDir.toString(), listOfFiles.get(0).getName())), serializedEntries.getCopy());
     }
 
+    @Test
+    public void testTableSegmentRecoveryCommandVersioned() throws Exception {
```

Comment:
does this test fail without the "unversioned" change?

Reply:
It does not fail with the changes.